Add Product Styling + Login Tweaks

doddsip opened the pull request:
styling for the add product button + tweaked login slightly
-adjusted login.html and login.css slightly:
---made input#username and input#password part of the same chunk of code
---made .username-label and .password part of the same chunk of code
---"WELCOME" header moved into the login form box
---sized adjusted because of this change

-the "new product" popup was stylized:
---a header was given to the popup, saying "ADD PRODUCT"
---labels were stylized
--input#product_name, input#inventory, input#price, input#unit_type, input#ideal_stock, input#donation, .add-category_id, button#add-product-label, button.btn-danger were all added to stylize the page
---add.html adjusted to deal with positioning

doddsip committed:
styling for the add product button + tweaked login slightly

-adjusted login.html and login.css slightly:
---made input#username and input#password part of the same chunk of code
---made .username-label and .password part of the same chunk of code
---"WELCOME" header moved into the login form box
---sized adjusted because of this change

-the "new product" popup was stylized:
---a header was given to the popup, saying "ADD PRODUCT"
---labels were stylized
--input#product_name, input#inventory, input#price, input#unit_type, input#ideal_stock, input#donation, .add-category_id, button#add-product-label, button.btn-danger were all added to stylize the page
---add.html adjusted to deal with positioning

doddsip committed:
Merge branch 'matthew-f-release' of https://github.com/423S25/repo1 into matthew-f-release

TheodoreBagley reviewed (approved):
Looks good to me. we need to change selenium testing for add product and add category with changes it seems

doddsip committed:
filter styling

-removed red background
-filter input, and "edit category" button were stylized
-to have all of this happen, main.html was changed slightly:
---removed the "Categories" label
---changed "All" to "Filters", styled that

doddsip committed:
category styling

-added "category.css"
-updated base.html to include this new file
-updated add_category.html in order to style it

doddsip committed:
Merge branch 'main' of https://github.com/423S25/repo1 into matthew-f-release

doddsip committed:
Revert "Merge branch 'main' of https://github.com/423S25/repo1 into matthew-f-release"

This reverts commit 59cc17099e41e383963557d810b892991f19f929, reversing
changes made to 7a3f3ebd9a765f99d00acff99fa65340fcbf0cd6.

TheQantas reviewed (commented):
The styling looks good. The target for the add product submit button needs to be changed for Selenium. It decided that `button:nth-child(22)` was an appropriate target for some reason, and with the redesign, the submit button is no longer the 22nd child. Changing the query selector in the `.side` file should fix it, I think.

doddsip committed:
Add Category & Add Product Tweaks

been having a lot of bugs but since things are seemingly going back on track (for me) i'll continue submitting what i have currently. will revert the revert if i REALLY need to. going to be hard to document all of what got changed from the revert but i'll document what i have now:

-login styling still there
-add product & add category styling still there
-filter styling still there

as far as im concerned any recent changes from main aren't here, from what i'm seeing. login will need to be fixed as soon as the merging begins.

now for this update:
-changed styling for the danger buttons of category & product adding. they are now a black X at the top left corner (have been unable to change the color as of comitting, may look into this if we need to)
---since this is the case in product.css/category.css the button.btn-danger have been moved and given their own styling.
---the "button" is completely invisible and should blend in with the background, with the user only seeing the icon.
---icon has a "active" state that inverts the color; a neat little thing to show that the user has clicked on it

-removed a duplicate color line in product.css in input#donation (redundant)
-removed the <head></head> tags between the styling reference for reports.html (also redundant)
-in product.css & category.css, the submit button has been given a hover state (turning from teal to green-yellow color)
-CRUD.side has been changed to compensate for the new css naming. should work now? i think? will change if it still doesn't work.

doddsip committed:
Merge branch 'main' into matthew-f-release

TheQantas committed:
Fixed selenium and submit button

TheQantas committed:
Further fix to selenium test
